cfme/infrastructure/pxe.py

```diff
@@ -591,6 +594,7 @@ class PXESystemImageTypesView(PXEMainView):
     """
     represents whole All System Image Types page
     """
+    table = Table('records_div')
```

Comment:
What's the purpose of this change in this PR?

Reply:
I included that in this PR mistakenly, thanks for highlighting it.